EnglishTranslation-master/QUEST.tsv

```diff
@@ -1128,25 +1128,25 @@ QUEST_20150317_001127	After that, Aiste also held out her bag and said, {nl}'Thi
-QUEST_20150317_001146	To be precise, what really turned his head toward the old woman, was what she held in her hands. {n]}Gold. The woman seemed to be obsessing over two gold pieces, doing something with them. {nl}Unable to hear her mumblings, Modestas drew closer to the strange woman.
+QUEST_20150317_001143	Whenever she could, Modestas' wife would often go to the shrine, and pray to the goddess' likeness. {nl}But Modestas continued his odd habits, and the people around him grew increasingly concerned. {nl}After all, the mind of a gambler would think, 'Today it may be small, but tomorrow it could be big!' {nl}And so the days rolled by, same as they ever were, until one peculiar day.
+QUEST_20150317_001144	Like most of his days, Modestas was heading home after gambling away his excess coins. {nl}He was walking through the Siauliai Woods on the path to Klaipeda, when he saw a strange, old woman on the road.
+QUEST_20150317_001145	This old woman seemed not only old and frail, but had an unnerving aura around her. {nl}She was acting strangely, as if deep in thought or as if she was obsessed with whatever she was doing. {nl}Modestas thought about simply ignoring the woman and walking past. Nothing would have happened if he did that, {nl}but curiosity got the better of him.
+QUEST_20150317_001146	To be precise, what really turned his head toward the old woman was what she held in her hands. {nl}Gold. The woman seemed to be obsessing over two gold pieces, doing something with them. {nl}Unable to hear her mumbling, Modestas drew closer to the strange woman.
 QUEST_20150317_001147	Over the jingling sound of gold, the old woman muttered. {nl}'If these two pieces of gold could ever marry, the baby gold would make me rich... but there's no life in these things...' {nl}Modestas heard those words, and decided to comment. {nl}'That gold won't make you rich by itself, so just put it into something worthwhile. {nl}Do you really think muttering at gold will do something?'{nl}
-QUEST_20150317_001148	The old woman spoke plainly, 'Why not? If this way bears fruit, it'll be safe and secure.' {nl}'That's ridiculous. You should do what I do and gamble.' {nl}The woman was incensed.
-QUEST_20150317_001149	So, you think your betting ways are better?' {nl}'Why wouldn't I? There are moments where I could win or lose, but your way won't make any gold at all.' {nl}With Modestas' words, the old woman paused in thought. Finally, she said, {nl}'Then let's have a bet.'
+QUEST_20150317_001148	The old woman spoke plainly, 'Why not? It's be safe and secure if this way bears fruit.' {nl}'That's ridiculous. You should do what I do and gamble.' {nl}The woman was incensed.
```

Comment:
Was the old woman's quote intentionally accented? It doesn't sound grammatically sound.

"Why not? If it works, this'll certainly be the most pleasant, risk-free way to wealth."